### PR TITLE
fix: publish group member info

### PR DESCRIPTION
**Pull request by mohsen-ghafouri**
Publish group member info just after group creation

related to status-mobile issue https://github.com/status-im/status-mobile/issues/17230

### CODE REVIEW

File: protocol/messenger_group_chat_test.go [hunk 1]
```diff
@@ -454,3 +455,48 @@ func (s *MessengerGroupChatSuite) TestGroupChatMembersRemovalOutOfOrder() {
...
+				return false
+			}
+			contact, ok := memberA.allContacts.Load(common.PubkeyToHex(&memberB.identity.PublicKey))
+			return ok && contact.DisplayName == "memberB"
+		},
+		"DisplayName is not the same",
+	)
+	s.Require().NoError(err)
+
+	defer s.NoError(admin.Shutdown())
```
Comment (cammellos) [inline protocol/messenger_group_chat_test.go]: no need to defer if you run it at the end of the file

**Review by cammellos — APPROVED** [review]

File: protocol/messenger_group_chat_test.go (hunk 1 above)
Comment (mohsen-ghafouri) [reply protocol/messenger_group_chat_test.go]: done

File: protocol/v1/membership_update_message.go [hunk 2]
```diff
@@ -448,6 +448,26 @@ func (g Group) Members() []string {
 	return g.members.List()
 }
 
+func (g Group) MemberPublicKeys() ([]*ecdsa.PublicKey, error) {
+	var publicKeys []*ecdsa.PublicKey
+	for _, memberPublicKey := range g.Members() {
+		publickKey, err := hexToPubkey(memberPublicKey)
```
Comment (ilmotta) [inline protocol/v1/membership_update_message.go]: Typo in `publickKey`.

File: protocol/v1/membership_update_message.go [hunk 3]
```diff
@@ -448,6 +448,26 @@ func (g Group) Members() []string {
 	return g.members.List()
 }
 
+func (g Group) MemberPublicKeys() ([]*ecdsa.PublicKey, error) {
+	var publicKeys []*ecdsa.PublicKey
```
Comment (ilmotta) [inline protocol/v1/membership_update_message.go]: It's a good idea to initialize with the capacity of `len(g.Members())`.

**Review by ilmotta — APPROVED** [review]

File: protocol/v1/membership_update_message.go (hunk 3 above)
Comment (mohsen-ghafouri) [reply protocol/v1/membership_update_message.go]: Sorry i didn't get that, can you give me an example?

Comment (mohsen-ghafouri) [reply protocol/v1/membership_update_message.go]: like this? @ilmotta 
```suggestion
	var publicKeys := make([]*ecdsa.PublicKey, 0, len(g.Members()))
```

Comment (ilmotta) [reply protocol/v1/membership_update_message.go]: Exactly @mohsen-ghafouri, if the max capacity of the slice is known, it's better to initialize it with that value.

Comment (mohsen-ghafouri) [reply protocol/v1/membership_update_message.go]: Thanks, done

**Pull request merged**